main.scss

```diff
@@ -0,0 +1,372 @@
+      }
+
+      25%,
+      44% {
+        transform: none;
+      }
+
+      50%,
+      60% {
+        transform: translateX(68px) translateY(-40px) skewX(5deg) skewY(2deg) scaleX(0.95);
+      }
+
+      66%,
+      100% {
+        transform: none;
+      }
+    }
+
+    @keyframes move-eye-skew2 {
+      0% {
+        transform: none;
+      }
+
+      20% {
+        transform: translateX(-80px) translateY(120px) scale(0.95);
+      }
+
+      25%,
+      34% {
+        transform: translateY(160px) scale(0.95);
+      }
+
+      40%,
+      50% {
+        transform: translateX(130px) translateY(160px) scale(0.95);
+      }
+
+      56%,
+      60% {
+        transform: translateX(80px) translateY(40px) scale(0.95);
+      }
+
+      70%,
+      80% {
+        transform: translateX(-120px) translateY(20px) scale(0.95);
+      }
+
+      85%,
+      100% {
+        transform: none;
+      }
+    }
+
+    .ball {
+      display: inline-block;
+      width: 100%;
+      height: 100%;
+      margin: 0;
+      border-radius: 50%;
+      position: relative;
+      background: radial-gradient(circle at 50% 40%, $secondary-color, $tertiary-color 66%, $primary-color 100%);
+
+      .shadow {
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        background: radial-gradient(circle at 50% 50%, rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.1) 40%, rgba(0, 0, 0, 0) 50%);
+        transform: rotateX(90deg) translateZ(-150px);
+        z-index: -1;
+      }
+    }
+
+    .iris {
+      width: 40%;
+      height: 40%;
+      margin: 30%;
+      border-radius: 50%;
+      background: radial-gradient(circle at 50% 50%, $text-color 0%, $primary-color 30%, $background-color 100%);
+      transform: translateX(68px) translateY(-60px) skewX(15deg) skewY(2deg);
+      position: absolute;
+      animation: move-eye-skew 5s ease-out infinite;
+
+      &:before {
+        content: "";
+        display: block;
+        position: absolute;
+        width: 37.5%;
+        height: 37.5%;
+        border-radius: 50%;
+        top: 31.25%;
+        left: 31.25%;
+        background: black;
+      }
+
+      &:after {
+        content: "";
+        display: block;
+        position: absolute;
+        width: 31.25%;
+        height: 31.25%;
+        border-radius: 50%;
+        top: 18.75%;
+        left: 18.75%;
+        background: rgba(255, 255, 255, 0.2);
+      }
+    }
+  }
+}
+
+.secondSection {
+  opacity: 0;
+  transition: opacity 1s ease-in-out;
+  transition-delay: 2s;
+
+  .question-container {
+    display: flex;
+    justify-content: center;
+    margin-top: 20px;
+    margin-bottom: 20px;
+  }
+
+  .form {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $text-color;
+    width: 430px;
+    border-radius: 8px;
+
+    .clearButton {
+      border-color: transparent;
+      background-color: transparent;
+      color: $primary-color;
+      font-weight: 200;
+    }
+
+    .Input {
```

Comment:
you use here upper-case name-convention. css styles should be either camelCase or kebab case.